src/Tasks.UnitTests/Copy_Tests.cs

```diff
@@ -2020,19 +2134,22 @@ public void CopyToDestinationFolderWithHardLinkFallbackNetwork()
             }
             catch (Exception)
             {
-                Console.WriteLine("CopyToDestinationFolderWithHardLinkFallbackNetwork test could not access the network.");
+                Console.WriteLine("CopyToDestinationFolderWithHardLinkFallbackNetwork test could not access the network as expected.");
```

Comment:
I don't think this is actually expected; I think it's here to hide "infrastructure" failures.

Reply:
Oops. Removed